admin-ui/app/company/[id]/page.tsx

```diff
@@ -1,36 +1,27 @@
 "use client";
-import { useQuery } from "@tanstack/react-query";
-import axios from "axios";
-import { useEffect, useState } from "react";
-import CompanyComponent from ".";
-import ICompany from "../company.model";
+import React, { useEffect, useState } from "react";
+import CompanyEditComponent from ".";
 
-const fetchData = async (id: string) => {
-  const res = await axios.get<ICompany>(
-    `http://localhost:8000/api/v1/companies/${id}`
-  );
-  return await res.data;
-};
+const _id = "641034daa9cd62cbc29a3099";
 
-export default function Page({
-  params,
-  searchParams,
-}: {
-  params: { id: string };
-  searchParams: { _id: string };
-}) {
-  const [company, setCompany] = useState<ICompany>();
-  const { data, error, isError, isLoading } = useQuery(
-    [`company-${params?.id}`],
-    async () => await fetchData(params?.id)
-  );
+const Page = () => {
+  const [company, setCompany] = useState([]);
 
-  if (isLoading) {
-    return <>Please wait .....</>;
-  }
-  if (isError) {
-    return <>Please wait there is some error</>;
-  }
+  const fetchData = async () => {
+    const users = await fetch(
```

Comment:
plz check, i think we have to use query, I think this 

 concurrentFeatures: true,

https://learnjsx.com/category/4/posts/nextjs-suspense